packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx

```diff
@@ -79,16 +88,15 @@ export class Facepile extends React.Component<IFacepileProps, {}> {
   }
 
   private _getElementWithoutOnClickEvent(personaControl: JSX.Element, persona: IFacepilePersona, index: number): JSX.Element {
-    return <div
+    return <button
```

Comment:
I think this may be wrong; what's the intention here? The whole reason there are two methods here is because an element without a click event should not be focusable. Otherwise the user expects it to be interactive; it seems like you're changing the logic here to render a button or... a button.

We *could*